documents/Strategies/Destroyment strategy.tex

```diff
@@ -21,13 +21,14 @@ \section{Introduction}
 To make room for construction or to 
 optimize a certain indicator."\\}
 
-There are two important parts in this user story that need to be discussed before implementing anything. The 'stuff' part: so which buildings need to be destroyed, and the 'benificial' part: so when buildings need to be destroyed. Again, these two questions will be discussed in the following two sections.
+There are two important parts in this user story that need to be discussed before implementing anything. The 'stuff' part: so which buildings need to be destroyed, and the 'benificial' part: so when buildings need to be destroyed. Again, these two questions will be discussed in the following two sections. 
 
 \section{When/why to destroy buildings}
-In this section the reasons to destroy a building will be explained. There are two types of reasons and both of them ultimately give a better indicator score. The first reason is to directly influence an indicator, and the second reason is to make space for a new building that influences an indicator. Both will be discussed in the following sections.
+In this section the reasons to destroy a building will be explained. There are two types of reasons and both of them ultimately give a better indicator score. The first reason is to directly influence an indicator, and the second reason is to make space for a new building that influences an indicator. Both will be discussed in the following sections. The agent needs to keep destroying buildings in order to build newer, higher buildings, until all indicators are at 100\%.
+
 
 \subsection{Direct influence}
-There is one direct reason to destroy a building. We created an indicator 'Destroy old buildings' that has the goal to destroy all pre-war type of buildings. There are quite a lot of these buildings, and if they are all destroyed this indicator will get to the 100\%. This is a reason to just destroy some buildings without needing another reason and it has a direct influence on an indicator. However, just destroying buildings had also a downside: it will possibly lower the other three indicators. These are to have a positive budget, build more education buildings and build high education buildings. So: if you destroy a certain building there is a chance it will impact to total score negatively. The only way to make it possitive is to build a building on that ground: see the following section.
```

Comment:
Is it really necassary to destroy 100% of the old buildings in order to get a 100% on one indicator?